components/common-data-items.js

```diff
@@ -47,6 +47,12 @@ export function DonorDataItems({ item, parents = null, children }) {
           <DataItemValue>{item.sex}</DataItemValue>
         </>
       )}
+      {item.virtual && (
+        <>
+          <DataItemLabel>Virtual</DataItemLabel>
+          <DataItemValue>{item.virtual ? "True" : "False"}</DataItemValue>
```

Comment:
Since the only value it can show is “True,” you can just put the string `True` there: `<DataItemValue>True</DataItemValue>`.